db/migrate/20220427155513_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+
+      t.timestamps
```

Comment:
- [ ] It would be better if you add the information needed in the users table. Currently, you only have the  `post_counter` column, kindly add the `name`, ` photo`, and `bio` columns to match the schema provided.

![Screenshot from 2022-04-21 14-09-34](https://user-images.githubusercontent.com/64173189/165585419-1b46b3fb-1047-4b13-94c8-be901020afce.png)